### PR TITLE
Creating my Profile in Bonganibendi.json

### PR DESCRIPTION
**Pull request by Bonganibendi**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2022-07-21 at 13 47 19

## Note to reviewers

I created a new fork for this project.

I fixed the errors from my previous pull request: #1514.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1515"></a>

### CODE REVIEW

**Comment by SohamRatnaparkhi** [comment]
Looks good to me! Thank you for contributing.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/Bonganibendi.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Bongani Bendile",
+  "type": "personal",
+  "bio": "I'm a finance and investment management student enthusiastic about the intersection of finance and software",
+  "avatar": "https://avatars.githubusercontent.com/u/58882596?s=400&u=85fa39e4c5bf28dcce96a22eae70f1a8e45565f2&v=4",
```
Comment (Panquesito7) [inline public/data/Bonganibendi.json:5]: It is best to use it this way, to have the link shorter and work better.
```suggestion
  "avatar": "https://github.com/Bonganibendi.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM 👍 
Thanks for contributing! 🚀

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**